service/worker/deletenamespace/deleteexecutions/workflow_test.go

```diff
@@ -52,16 +52,14 @@ func Test_DeleteExecutionsWorkflow_Success(t *testing.T) {
 	testSuite := &testsuite.WorkflowTestSuite{}
 	env := testSuite.NewTestWorkflowEnvironment()
 
-	var a *Activities
```

Comment:
This was a way to say "`a` is a `nil`, I know this, and I am not using it besides for reflection. If anyone tries to use it's value it should panic".

Reply:
I suggest to keep these `nil` local vars instead of 
```
var activityHandles = &Activities{}
var localActivityHandles = &LocalActivities{}
```

Reply:
and if I get it right, you don't need any of these constructors at all.

Reply:
Are you suggesting I change it back? I thought `var activityHandles = &Activities{}` also made that clear.